src/System.Management.Automation/engine/ExperimentalFeature/ExperimentalFeature.cs

```diff
@@ -168,6 +168,14 @@ static ExperimentalFeature()
             }
             catch (Exception e) when (LogException(e)) { }
 
+            if (enabledFeatures.Length == 0 && !string.IsNullOrEmpty(PSVersionInfo.PSCurrentVersion.PreReleaseLabel))
+            {
+                // if no experimental features are configured in pwsh.config.json file
+                // and pwsh version is preview
+                // then we enable all experimental features by default.
+                enabledFeatures = engineFeatures.Select(static f => f.Name).ToArray();
```

Comment:
This won't work. Modules can declare experimental features which are not in `engineFeatures`.

Reply:
For in-box modules they are in engineFeatures.
For out-box modules it should be up to the authors of these modules to decide if they want to include experimental features by default.

Reply:
No, we have experimental features declared in `Utility` modules, even today.
```
PS:2> Get-ExperimentalFeature | % Source
PSEngine
PSEngine
PSEngine
PSEngine
PSEngine
PSEngine
C:\program files\powershell\7\Modules\Microsoft.PowerShell.Utility\Microsoft.PowerShell.Utility.psd1
C:\program files\powershell\7\Modules\Microsoft.PowerShell.Utility\Microsoft.PowerShell.Utility.psd1
```

Reply:
Why not have them in engineFeatures?

Reply:
Because
1. it's not necessary from the developer's perspective -- the module manifest already needs to have the experimental feature declared.
2. it should be the same as any other module and exercise the same discovery code path.

We are not going to add a burden to the developer just for making the build easier, especially given that the build changes are already done.